articles/app-service/app-service-hybrid-connections.md

```diff
@@ -22,160 +22,165 @@ ms.author: ccompy
 
-The HCM has a UI to configure it.  After the HCM is installed you can bring up the UI by running the HybridConnectionManagerUi.exe that sits in the Hybrid Connection Manager installation directory.  It is also easily reached on Windows 10 by typing *Hybrid Connection Manager UI* in your search box.  
+The HCM has a UI to configure it.  After the HCM is installed, you can bring up the UI by running the HybridConnectionManagerUi.exe that sits in the Hybrid Connection Manager installation directory.  It is also easily reached on Windows 10 by typing *Hybrid Connection Manager UI* in your search box.  
 
-When the HCM UI is started, the first thing you see is a table that lists all of the hybrid connections that are configured with this instance of the HCM.  If you wish to make any changes you will need to authenticate with Azure. 
+When the HCM UI is started, the first thing you see is a table that lists all of the Hybrid Connections that are configured with this instance of the HCM.  If you wish to make any changes, you will need to authenticate with Azure. 
 
 To add one or more Hybrid Connections to your HCM:
 
 1. Start the HCM UI
-1. Click Configure another hybrid connection
+1. Click Configure another Hybrid Connection
 ![][8]
 
 1. Sign in with your Azure account
 1. Choose a subscription
-1. Click on the hybrid connections you want this HCM to relay
+1. Click on the Hybrid Connections you want the HCM to relay
 ![][9]
 
 1. Click Save
 
-At this point you will see the hybrid connections you added.  You can also click on the configured hybrid connection and see details about the connection.
+At this point, you will see the Hybrid Connections you added.  You can also click on the configured Hybrid Connection and see details about the it.
 
 ![][10]
 
-For your HCM to be able to support the hybrid connections it is configured with, it needs:
+For your HCM to be able to support the Hybrid Connections it is configured with, it needs:
 
 - TCP access to Azure over ports 80 and 443
-- TCP access to the hybrid connection endpoint
-- ability to do DNS look ups on the endpoint host and the azure servicebus namespace
+- TCP access to the Hybrid Connection endpoint
+- Ability to do DNS look-ups on the endpoint host and the Azure Service Bus namespace
 
-The HCM supports both new hybrid connections as well as the older BizTalk hybrid connections.
+The HCM supports both new Hybrid Connections, as well as the older BizTalk Hybrid Connections.
+
+> [!NOTE]
+> Azure Relay relies on Web Sockets for connectivity. Because Windows Server 2008 R2 does not support Web Sockets, you cannot use the new Hybrid Connections if the HCM is installed on Windows Server 2008 R2. You will need to install the HCM on Windows Server 2012 or later if you want to use Hybrid Connections that use Azure Relay.
```

Comment:
Azure Relay relies on Web Sockets for connectivity.  This capability is only available on Windows Server 2012 or newer.  Because of that the Hybrid Connection Manager is not supported on anything earlier than Windows Server 2012.